conductor-client/src/main/java/com/openlattice/postgres/PostgresTable.java

```diff
@@ -299,6 +301,11 @@
                             ANALYZER,
                             MULTI_VALUED,
                             INDEX_TYPE );
+    public static final PostgresTableDefinition EXTERNAL_ROLES =
```

Comment:
Might be less confusing to call these like EXTERNAL_PERMISSION_ROLES so as to not confuse them with any roles in external databases